graphql/graphql.go

```diff
@@ -468,6 +482,40 @@ func (t *Transaction) CumulativeGasUsed(ctx context.Context) (*hexutil.Uint64, e
 	return &ret, nil
 }
 
+func (t *Transaction) BlobGasUsed(ctx context.Context) (*hexutil.Uint64, error) {
+	tx, _ := t.resolve(ctx)
+	if tx == nil {
+		return nil, nil
+	}
+	if tx.Type() != types.BlobTxType {
+		return nil, nil
+	}
+
+	receipt, err := t.getReceipt(ctx)
+	if err != nil || receipt == nil {
+		return nil, err
+	}
+	ret := hexutil.Uint64(receipt.BlobGasUsed)
+	return &ret, nil
+}
+
+func (t *Transaction) BlobGasPrice(ctx context.Context) (*hexutil.Big, error) {
```

Comment:
This seems to be the same as maxFeePerBlobGas? at least here is being set this way: https://github.com/ethereum/go-ethereum/blob/5cf53f51ac556cfff2aee9d405efd336298a3aca/core/state_processor.go#L139

But not sure why the same field would be repeated in receipt and tx objects

Reply:
Ok no the receipt field is different. The line I shared might be a bug

Reply:
Haha, seems fixed in #28082